### PR TITLE
[zh-cn]: update the translation of SVG `<feflood>` and `<fefunc*>` element

### PR DESCRIPTION
**Pull request by T34-active**
### Description

* https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feFlood
* https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feFuncA
* https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feFuncB
* https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feFuncG
* https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feFuncR

#### `<feflood>`

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/feflood/index.md
* Last commit: https://github.com/mdn/content/commit/3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1

#### `<feFuncA>`

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/fefunca/index.md
* Last commit: https://github.com/mdn/content/commit/3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1

#### `<feFuncB>`

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/fefuncb/index.md
* Last commit: https://github.com/mdn/content/commit/3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1

#### `<feFuncG>`

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/fefuncg/index.md
* Last commit: https://github.com/mdn/content/commit/3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1

#### `<feFuncR>`

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/fefuncr/index.md
* Last commit: https://github.com/mdn/content/commit/3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1

### CODE REVIEW

File: files/zh-cn/web/svg/element/feflood/index.md [hunk 1]
```diff
@@ -1,56 +1,80 @@
...
+title: <feFlood>
 slug: Web/SVG/Element/feFlood
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
 ---
 
 {{SVGRef}}
 
-该滤镜用{{ SVGAttr("flood-color") }}元素定义的颜色和{{ SVGAttr("flood-opacity") }}元素定义的不透明度填充了滤镜子区域。
+**`<feFlood>`** SVG 滤镜基元使用由 {{SVGAttr("flood-color")}} 和 {{SVGAttr("flood-opacity")}} 定义的颜色和透明度填充滤镜子区域。
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/feflood/index.md:10]: 滤镜原语，我在以前的 PR 中提到过

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/svg/element/feflood/index.md (hunk 1 above)
Comment (T34-active) [inline files/zh-cn/web/svg/element/feflood/index.md:10]:
```suggestion
**`<feFlood>`** SVG 滤镜原语使用由 {{SVGAttr("flood-color")}} 和 {{SVGAttr("flood-opacity")}} 定义的颜色和透明度填充滤镜子区域。
```

File: files/zh-cn/web/svg/element/fefunca/index.md [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
+title: <feFuncA>
 slug: Web/SVG/Element/feFuncA
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
 ---
 
 {{SVGRef}}
 
-[SVG](/zh-CN/docs/Web/SVG) 滤镜 **`<feFuncA>`** 为它的父元素 {{SVGElement("feComponentTransfer")}} 的输入图形的透明度（alpha）组件定义了变换函数。
+**`<feFuncA>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜基元为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的 alpha 分量定义了传递函数。
```
Comment (T34-active) [inline files/zh-cn/web/svg/element/fefunca/index.md:10]:
```suggestion
**`<feFuncA>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜原语为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的 alpha 分量定义了传递函数。
```

File: files/zh-cn/web/svg/element/fefuncb/index.md [hunk 3]
```diff
@@ -1,39 +1,40 @@
...
+title: <feFuncB>
 slug: Web/SVG/Element/feFuncB
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
 ---
 
 {{SVGRef}}
 
-该滤镜为它的父{{SVGElement("feComponentTransfer")}}元素的输入图形的蓝色成分定义了变换函数。
+**`<feFuncB>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜基元为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的蓝色分量定义了传递函数。
```
Comment (T34-active) [inline files/zh-cn/web/svg/element/fefuncb/index.md:10]:
```suggestion
**`<feFuncB>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜原语为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的蓝色分量定义了传递函数。
```

File: files/zh-cn/web/svg/element/fefuncg/index.md [hunk 4]
```diff
@@ -1,39 +1,40 @@
...
+title: <feFuncG>
 slug: Web/SVG/Element/feFuncG
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
 ---
 
 {{SVGRef}}
 
-该滤镜为它的父{{SVGElement("feComponentTransfer")}}元素的输入图形的绿色成分定义了变换函数。
+**`<feFuncG>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜基元为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的绿色分量定义了传递函数。
```
Comment (T34-active) [inline files/zh-cn/web/svg/element/fefuncg/index.md:10]:
```suggestion
**`<feFuncG>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜原语为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的绿色分量定义了传递函数。
```

File: files/zh-cn/web/svg/element/fefuncr/index.md [hunk 5]
```diff
@@ -1,39 +1,37 @@
...
+title: <feFuncR>
 slug: Web/SVG/Element/feFuncR
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
 ---
 
 {{SVGRef}}
 
-该滤镜为它的父{{SVGElement("feComponentTransfer")}}元素的输入图形的红色成分定义了变换函数。
+**`<feFuncR>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜基元为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的红色分量定义了传递函数。
```
Comment (T34-active) [inline files/zh-cn/web/svg/element/fefuncr/index.md:10]:
```suggestion
**`<feFuncR>`** [SVG](/zh-CN/docs/Web/SVG) 滤镜原语为其 {{SVGElement("feComponentTransfer")}} 父元素输入图形的红色分量定义了传递函数。
```

**Review by T34-active — COMMENTED** [review]
"基元" replaces "原语"

File: files/zh-cn/web/svg/element/feflood/index.md [hunk 6]
```diff
@@ -1,56 +1,80 @@
...
-- {{ SVGElement("feGaussianBlur") }}
-- {{ SVGElement("feImage") }}
-- {{ SVGElement("feMerge") }}
-- {{ SVGElement("feMorphology") }}
-- {{ SVGElement("feOffset") }}
-- {{ SVGElement("feSpecularLighting") }}
-- {{ SVGElement("feTile") }}
-- {{ SVGElement("feTurbulence") }}
-- [SVG 教程：滤镜效果](/zh-CN/SVG/Tutorial/Filter_effects)
+- [SVG 滤镜属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/feflood/index.md:61]:
```suggestion
- [SVG 滤镜原语属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```

File: files/zh-cn/web/svg/element/fefunca/index.md [hunk 7]
```diff
@@ -36,11 +27,14 @@ _无。_
...
 
-- {{ SVGElement("filter") }}
-- {{ SVGElement("animate") }}
-- {{ SVGElement("set") }}
-- {{ SVGElement("feComponentTransfer")}}
-- {{ SVGElement("feFuncR") }}
-- {{ SVGElement("feFuncB") }}
-- {{ SVGElement("feFuncG") }}
-- [SVG 教程：滤镜效果](/zh-CN/SVG/Tutorial/Filter_effects)
+- [SVG 滤镜属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/fefunca/index.md:30]:
```suggestion
- [SVG 滤镜原语属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```

File: files/zh-cn/web/svg/element/fefuncb/index.md [hunk 8]
```diff
@@ -1,39 +1,40 @@
...
 
-- {{ SVGElement("filter") }}
-- {{ SVGElement("animate") }}
-- {{ SVGElement("set") }}
-- {{ SVGElement("feComponentTransfer") }}
-- {{ SVGElement("feFuncA") }}
-- {{ SVGElement("feFuncR") }}
-- {{ SVGElement("feFuncG") }}
-- [SVG 教程：滤镜效果](/zh-CN/SVG/Tutorial/Filter_effects)
+- [SVG 滤镜属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/fefuncb/index.md:30]:
```suggestion
- [SVG 滤镜原语属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```

File: files/zh-cn/web/svg/element/fefuncb/index.md [hunk 9]
```diff
@@ -1,39 +1,40 @@
...
+- {{SVGAttr('flood-color')}} 属性
+- {{SVGAttr('flood-opacity')}} 属性
+- {{SVGElement("filter")}}
+- {{SVGElement("animate")}}
+- {{SVGElement("set")}}
+- {{SVGElement("feComponentTransfer")}}
+- {{SVGElement("feFuncR")}}
+- {{SVGElement("feFuncB")}}
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/fefuncb/index.md:38]:
```suggestion
- {{SVGElement("feFuncA")}}
- {{SVGElement("feFuncR")}}
```

File: files/zh-cn/web/svg/element/fefuncg/index.md [hunk 10]
```diff
@@ -1,39 +1,40 @@
...
 
-- {{ SVGElement("filter") }}
-- {{ SVGElement("animate") }}
-- {{ SVGElement("set") }}
-- {{ SVGElement("feComponentTransfer") }}
-- {{ SVGElement("feFuncA") }}
-- {{ SVGElement("feFuncR") }}
-- {{ SVGElement("feFuncB") }}
-- [SVG 教程：滤镜效果](/zh-CN/SVG/Tutorial/Filter_effects)
+- [SVG 滤镜属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/fefuncg/index.md:30]:
```suggestion
- [SVG 滤镜原语属性](/zh-CN/docs/Web/SVG/Attribute#滤镜属性)
```

File: files/zh-cn/web/svg/element/fefuncg/index.md [hunk 11]
```diff
@@ -1,39 +1,40 @@
...
+- {{SVGAttr('flood-color')}} 属性
+- {{SVGAttr('flood-opacity')}} 属性
+- {{SVGElement("filter")}}
+- {{SVGElement("animate")}}
+- {{SVGElement("set")}}
+- {{SVGElement("feComponentTransfer")}}
+- {{SVGElement("feFuncR")}}
+- {{SVGElement("feFuncB")}}
+- {{SVGElement("feFuncG")}}
```
Comment (yin1999) [inline files/zh-cn/web/svg/element/fefuncg/index.md:39]:
```suggestion
- {{SVGElement("feFuncA")}}
- {{SVGElement("feFuncR")}}
- {{SVGElement("feFuncB")}}
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]

**Pull request merged**